acsys/__init__.py

```diff
@@ -694,6 +694,15 @@ async def request_stream(self, remtsk, message, *, proto=None, timeout=1000):
 isn't an integer, ValueError is raised.
 
         """
+
+        pending = set([])
+
+        # If the caller provides a "signalling" future, wrap it in a
+        # `task` (required by `asyncio.wait()`.)
+
+        if done_fut is not None:
+            pending.add(asyncio.create_task(done_fut))
```

Comment:
`.create_task` expects a coroutine but `done_fut` is a future.
I'm getting `TypeError: a coroutine was expected, got <Future pending>`